modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/tx/ClientTransactionBeginRequest.java

```diff
@@ -113,11 +76,6 @@ public class ClientTransactionBeginRequest {
             long resourceId = resources.put(new ClientResource(tx, tx::rollbackAsync));
             out.packLong(resourceId);
 
-            if (enableDirectMapping && !readOnly) {
```

Comment:
Looks like a breaking change on protocol level, should we check the new feature flag?

Reply:
I've added the commit [1] which ensures this code path will only be called in proxy mode.
So it's safe to remove all code related to direct mapping from here.
[1] https://github.com/apache/ignite-3/pull/5853/commits/02fa047ff55e8a7d97f52bbc2653eb3a8af74328